CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+This document outlines the process to get your pull request accepted.
+
+## Start With An Issue
+
+Prior to creating a pull request it is a good idea to [create an issue].
+This is especially true if the change request is something large.
+The bug, feature request, or other type of issue can be discussed prior to
+creating the pull request. This can reduce rework.
+
+[create an issue]: https://github.com/rancher-sandbox/rancher-desktop/issues/new
+
+## Sign Your Commits
+
+A sign-off is a line at the end of the explanation for a commit.
+All commits must to be signed. Your signature certifies that you wrote the patch
+or otherwise have the right to contribute the material. When you sign-off you
+agree to the following rules
+(from [developercertificate.org](https://developercertificate.org/)):
+
+```
+Developer Certificate of Origin
+Version 1.1
+
+Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
+1 Letterman Drive
+Suite D4700
+San Francisco, CA, 94129
+
+Everyone is permitted to copy and distribute verbatim copies of this
+license document, but changing it is not allowed.
+
+Developer's Certificate of Origin 1.1
+
+By making a contribution to this project, I certify that:
+
+(a) The contribution was created in whole or in part by me and I
+    have the right to submit it under the open source license
+    indicated in the file; or
+
+(b) The contribution is based upon previous work that, to the best
+    of my knowledge, is covered under an appropriate open source
+    license and I have the right under that license to submit that
+    work with modifications, whether created in whole or in part
+    by me, under the same open source license (unless I am
+    permitted to submit under a different license), as indicated
+    in the file; or
+
+(c) The contribution was provided directly to me by some other
+    person who certified (a), (b) or (c) and I have not modified
+    it.
+
+(d) I understand and agree that this project and the contribution
+    are public and that a record of the contribution (including all
+    personal information I submit with it, including my sign-off) is
+    maintained indefinitely and may be redistributed consistent with
+    this project or the open source license(s) involved.
+```
+
+Then you add a line to every git commit message:
+
+    Signed-off-by: Joe Smith <joe.smith@example.com>
+
+Use your real name (sorry, no pseudonyms or anonymous contributions).
+
+If you set your `user.name` and `user.email` git configs, you can sign your
+commit automatically with `git commit -s`.
+
+Note: If your git config information is set properly then viewing the `git log`
+information for your commit will look something like this:
+
+```
+Author: John Smith <john.smith@example.com>
+Date:   Thu Feb 2 11:41:15 2018 -0800
+
+    Update README
+
+    Signed-off-by: John Smith <john.smith@example.com>
+```
+
+Notice the `Author` and `Signed-off-by` lines match. If they don't your PR will
```

Comment:
```suggestion
Notice how the `Author` and `Signed-off-by` lines match. If they don't your PR will
```

Question: Does epinio have this automated DC0 check ?

Reply:
Can we commit with this language and add the DC0 check next week? I'm asking Matt how they set this up.